Formula/f/flight_sql.rb

```diff
@@ -0,0 +1,46 @@
+class FlightSql < Formula
+  desc "Apache Arrow Flight SQL Server - with DuckDB and SQLite back-ends"
+  homepage "https://arrow.apache.org/docs/format/FlightSql.html"
+  url "https://github.com/voltrondata/flight-sql-server-example.git",
+      tag:      "v1.2.1",
+      revision: "6280bfb7642ac640f6352f01081a7a7b1805a318"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gflags" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@3" => :build
```

Comment:
I'm very surprised this is not a runtime dependency. Does it make a static build?

Reply:
You shouldn't since that requires a recompile for security fixes

Reply:
We prefer a static build to make it easy to install for non-brew users for the time being.  Does this prevent us from using brew as a distribution method?

Reply:
Yes, just like most other package managers we prefer dynamic linking since that will not require us to recompile all builds when there are openssl security updates.

Reply:
I understand, but I'm adopting a strategy similar to DuckDB which also statically links OpenSSL - how did they get approved?

Reply:
DuckDB shouldn't have been added with static links to OpenSSL. But reviewers are human and can miss things.